Sources/DVR/Filter.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  Filter.swift
+//  DVR
+//
+//  Created by Jáir Myree on 6/18/21.
+//  Copyright © 2021 Venmo. All rights reserved.
+//
```

Comment:
This will need to change to use the Microsoft license header. If this is their stock header it's a little weird since they specify an MIT license: https://github.com/venmo/DVR/blob/master/LICENSE

Reply:
This is interesting, that was autogenerated so it is definitely weird that's how it came out. I'll replace it with the Microsoft header.

Reply:
It looks like the DVR source generally has no header, so I'd say stick to that and simply remove this, assuming we want to merge this back to Venmo eventually.